inc/functions.inc

```diff
@@ -9562,20 +9562,28 @@ if (!function_exists("livesearch_tuotehaku")) {
         // Katsotaan mist� normaalivarastoista k�ytt�j� saa myyd�
         if (in_array($yhtiorow['livetuotehaku_hakutapa'], array('E', 'G', 'I', 'K', 'M'))) {
 
-          $varastolisa = $kukarow['varasto'] != '' ? "AND tunnus IN ({$kukarow['varasto']})" : "";
+          if (!empty($laskurow['varasto'])) {
+            $laskurow['varasto'] = (int) $laskurow['varasto'];
+            $varastolisa = " AND tunnus IN ({$laskurow["varasto"]})";
+          }
+          else {
+            $varastolisa = " AND tyyppi  = '' ";
+            $varastolisa .= $kukarow['varasto'] != '' ? " AND tunnus IN ({$kukarow['varasto']}) " : "";
+          }
 
           $query = "SELECT GROUP_CONCAT(DISTINCT tunnus) AS varastotunnukset
                     FROM varastopaikat
                     WHERE yhtio = '{$kukarow['yhtio']}'
-                    AND tyyppi  = ''
+                    AND tyyppi != 'P'
```

Comment:
@tomit @samiluukkonen Jos myy poistetusta varastosta, niin saldoa ei näytetä? Onko hyvä näin vai oisko parempi näyttää poistetunkin varaston saldot jos ollaan myymässä poistetusta varastosta?